doc/internal/man3/ossl_provider_new.pod

```diff
@@ -84,14 +86,20 @@ To indicate a built-in provider, the C<init_function> argument must
 point at the provider initialization function for that provider.
 
 ossl_provider_free() decrements a I<provider object>'s reference
-counter; if it drops to one, the I<provider object> will be
-inactivated (it's teardown function is called) but kept in the store;
-if it drops down to zero, the associated module will be unloaded if
-one was loaded, and the I<provider object> will be freed.
+counter; if it drops below 2, the I<provider object> is assumed to
+have fallen out of use and will be inactivated (it's teardown function
```

Comment:
_its_ not _it's_.  Deactivated rather than inactivated.

Reply:
Will fix

Reply:
(found another one too)

Reply:
Found another one too